akka-http-core/src/test/scala/akka/http/scaladsl/model/headers/HeaderSpec.scala

```diff
@@ -121,6 +121,8 @@ class HeaderSpec extends FreeSpec with Matchers {
         Upgrade(Vector(UpgradeProtocol("HTTP", Some("2.0")))),
         `User-Agent`("Akka HTTP Client 2.4"),
         `X-Forwarded-For`(RemoteAddress(InetAddress.getByName("192.168.0.1"))),
+        `X-Forwarded-Host`(RemoteAddress(InetAddress.getByName("192.168.0.2"))),
+        `X-Forwarded-Proto`("https"),
```

Comment:
Would be nice to also add additional test cases in akka-http-core/src/test/scala/akka/http/impl/model/parser/HttpHeaderSpec.scala that also validate rendering when the passed `RemoteAddress` has a port.

Reply:
I'll add some tests for the new headers